### PR TITLE
DOCS: Move Nima files to SE and remove Nima references

### PR DESCRIPTION
**Pull request by ljamen**
Addresses #7289 

Sitegen.yaml updates included. Leaving injection in docs for now.

### CODE REVIEW

File: docs/se/testing.adoc [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+<5> Call the `client` to obtain server response
+<6> Perform the necessary assertions.
+
+To trigger the framework to start and configure the Níma server, annotate the testing class with the `@ServerTest` annotation.
+
+In this test, the `Http1Client` client is used, which means that the framework will create, configure, and inject this object as a parameter to the constructor.
+
+To set up routing, a static method annotated with `@SetUpRoute` is present. The framework uses this method to inject the configured routing to the subject of testing – in the current case, the `Quickstart` application.
+
+As everything above is performed by the testing framework, regular unit tests can be done. After completing all tests, the testing framework will shut down the Níma server.
```
Comment (Geetha-Savithriamma) [inline docs/se/testing.adoc:206]: ... framework will shut down the Níma server. (we should remove ref here?)

File: docs/nima/inject/injection_intro.adoc [hunk 2]
```diff
@@ -66,7 +76,7 @@ Once you have planned how each of your resources should support Injection, you s
 
 == API
 
-Adding Ijection behavior to your Helidon Níma application involves just a few simple steps.
+Adding Injection behavior to your Helidon Níma application involves just a few simple steps.
```
Comment (Geetha-Savithriamma) [inline docs/nima/inject/injection_intro.adoc:79]: Helidon Níma application (remove Nima ref here?)

File: docs/se/inject/injection_intro.adoc [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+== Usage
+
+Once you have planned how each of your resources should support Injection, you specify the behavior in one of two ways:
+
+
+
+== API
+
+Adding Injection behavior to your Helidon Níma application involves just a few simple steps.
```
Comment (Geetha-Savithriamma) [inline docs/se/inject/injection_intro.adoc:79]: ... your Helidon Níma application (remove Nima ref here?)

File: docs/se/testing.adoc [hunk 4]
```diff
@@ -0,0 +1,393 @@
...
+|===
+
+In addition, a static method annotated with `@SetUpServer` can be defined for `@ServerTest`, which has a single parameter of `WebServer.Builder`.
+
+The following table lists the injectable types (through constructor or method injection).
+
+- Type - type that can be injected
+- Socket - if checked, you can use the `@Socket` annotation to obtain a value specific to that named socket
+- Annotation - which annotations support this injection
+- Modules - which Níma extension modules support this injection
```
Comment (Geetha-Savithriamma) [inline docs/se/testing.adoc:108]: ... your Helidon Níma application (remove Nima ref here?)

File: docs/se/testing.adoc [hunk 5]
```diff
@@ -0,0 +1,393 @@
...
+}
+----
+<1> Use `@ServerTest` to trigger the testing framework.
+<2> Inject `Http1Client` for the test.
+<3> SetUp routing for the test.
+<4> Regular `JUnit` test method.
+<5> Call the `client` to obtain server response
+<6> Perform the necessary assertions.
+
+To trigger the framework to start and configure the Níma server, annotate the testing class with the `@ServerTest` annotation.
```
Comment (Geetha-Savithriamma) [inline docs/se/testing.adoc:200]: configure the Níma server, ... (remove Nima ref here?)

File: docs/se/testing.adoc [hunk 6]
```diff
@@ -0,0 +1,393 @@
...
+
+In this test, the `Http1Client` client is used, which means that the framework will create, configure, and inject this object as a parameter to the constructor.
+
+To set up routing, a static method annotated with `@SetUpRoute` is present. The framework uses this method to inject the configured routing to the subject of testing – in the current case, the `Quickstart` application.
+
+As everything above is performed by the testing framework, regular unit tests can be done. After completing all tests, the testing framework will shut down the Níma server.
```
Comment (Geetha-Savithriamma) [inline docs/se/testing.adoc:206]: shut down the Níma server. (remove Nima ref here?)

File: docs/se/testing.adoc [hunk 7]
```diff
@@ -0,0 +1,393 @@
...
+    <groupId>io.helidon.testing.junit5</groupId>
+    <artifactId>helidon-testing-junit5-websocket</artifactId>
+    <scope>test</scope>
+</dependency>
+----
+
+
+=== WebSocket Testing Example
+
+The WebSocket Testing extension adds support for routing configuration and injection of WebSocket related artifacts, such as WebSockets and DirectWsClient in Helidon Níma unit tests.
```
Comment (Geetha-Savithriamma) [inline docs/se/testing.adoc:271]: in Helidon Níma unit tests. (remove Nima ref here?)

**Review by Geetha-Savithriamma — COMMENTED** [review]
Added a few comments.

File: docs/nima/inject/injection_intro.adoc (hunk 2 above)
Comment (ljamen) [reply docs/nima/inject/injection_intro.adoc:79]: Fixed in the SE version. The Nima directory will be deleted so changes are made to the SE versions only.

File: docs/nima/inject/injection_intro.adoc
Comment (barchetta) [inline docs/nima/inject/injection_intro.adoc:1]: To reduce confusion how about we remove the `nima` directory now, in this PR.

File: docs/sitegen.yaml
Comment (barchetta) [inline docs/sitegen.yaml:1]: Can we add back the top level "Get Started" in the navbar?  I know we now have the very nice "Getting Started" tile in the overview, but I like also having  "Get Started"  visible in the navbar with zero clicks needed to see it:
```
        - type: "PAGE"
          title: "Get Started"
          source: "about/prerequisites.adoc"
          glyph:
            type: "icon"
            value: "rocket_launch"
```

File: docs/nima/inject/injection_intro.adoc
Comment (ljamen) [reply docs/nima/inject/injection_intro.adoc:1]: Removed Nima dir and docs

File: docs/sitegen.yaml
Comment (ljamen) [reply docs/sitegen.yaml:1]: Added it back in and working on content for it

**Comment by ljamen** [comment]
Replaced by #7347

**Pull request closed without merging**